src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -377,8 +377,9 @@ and `CustomPainter`, the latter of which implements your algorithm to draw to
 the canvas.
 
 To learn how to implement a signature painter in Flutter, see Collin's answer on
-[StackOverflow](https://stackoverflow.com/questions/46241071/create-signature-area-
-for-mobile-app-in-dart-flutter).
```

Comment:
@sfshaza2, here is an example of a link that is broken by word-wrapping a paragraph containing a link with an embedded URL. It is usually best to create the link definition at the page bottom, or just below the paragraph. In that way, word wrapping won't have this nasty side-effect.